packages/turbo-vsc/src/extension.ts

```diff
@@ -42,6 +42,8 @@ const decoration = window.createTextEditorDecorationType({
   color: "#04f1f9", // something like cyan
 });
 
+const logs = window.createOutputChannel("Turbo");
```

Comment:
Nit: if this is user-facing in the VSCode UI, we might want either "turbo" like the command, or "Turborepo" like we use in our docs

Reply:
I have changed this to be "Turborepo Extension". Rust analyzer calls their vscode-side part "Rust Analyzer Client" but our extension has a little more to it than just an LSP client :)